src/utils/mlx_list.c

```diff
@@ -18,8 +18,9 @@ int32_t mlx_lstsize(mlx_list_t* lst)
 {
 	int32_t	i = 0;
 
-	if (!lst)
+	/*if (!lst)
 		return (i);
+	this check isn't necessary since the while loop won't execute when lst is NULL*/
```

Comment:
If thats the case just remove the section of code as well as the comment.

Reply:
coming up